packages/wdio-browserstack-service/src/util.ts

```diff
@@ -1294,18 +1294,28 @@ export const patchConsoleLogs = o11yErrorHandler(() => {
     const BSTestOpsPatcher = new logPatcher({})
 
     Object.keys(consoleHolder).forEach((method: keyof typeof console) => {
+        if (!(method in console) || typeof console[method] !== 'function') {
+            BStackLogger.debug(`Skipping method: ${method}, exists: ${method in console}, type: ${typeof console[method]}`)
+            return
+        }
         // @ts-expect-error
         const origMethod = console[method].bind(console)
```

Comment:
This shouldn't be needed anymore.
```suggestion
        const origMethod = console[method].bind(console)
```

Reply:
@christian-bromann,
After removing // @ts-expect-error I encountered the following error when running pnpm run setup:

Error: src/util.ts(1301,44): error TS2349: This expression is not callable.
  Each member of the union type '{ <T>(this: T, thisArg: ThisParameterType<T>): OmitThisParameter<T>; <T, A extends any[], B extends any[], R>(this: (this: T, ...args: [...A, ...B]) => R, thisArg: T, ...args: A): (...args: B) => R; } | { ...; }' has signatures, but none of those signatures are compatible with each other.
For now, I have added it back.